src/ngTouch/touch.js

```diff
@@ -11,6 +11,13 @@
  *
  * See {@link ngTouch.$swipe `$swipe`} for usage.
  *
+ * @deprecated
+ * sinceVersion="1.7.0"
+ * The ngTouch module with the {@link ngTouch.$swipe `$swipe`} service and
+ * the {@link ngTouch.ngSwipeLeft} and {@link ngTouch.ngSwipeRight} directives are
+ * deprecated. Instead, stand-alone libraries for touch handling and gesture interaction
+ * should be used, for example [HammerJS](https://hammerjs.github.io/) (which is also used by
+ * Angular).
```

Comment:
This might be confusing to people not aware of the AngularJS-vs-Angular distinction.
Maybe add a link to Angular or make it more explicit that you refer to the newer version (or both :grin:).

Reply:
But the naming change was a pretty big deal so I think it's okay.